web/wsgiserver/wsgiserver2.py

```diff
@@ -1016,7 +1016,7 @@ def sendall(self, data):
             try:
                 bytes_sent = self.send(data)
                 data = data[bytes_sent:]
-            except socket.error, e:
+            except (socket.error, e):
```

Comment:
In py3, It must be `except socket.error as e`(works in py2 too), not `except (socket.error, e)`. 
`except (socket.error, e)` means except two Exception: socket.error and e, but e is an instance of `socket.error`.